.azure-devops/prbuild/mlz-pr-sbom-pipelines.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# disable CI per:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#disabling-the-ci-trigger
+trigger: none
+
+pr:
+  branches:
+    include:
+    - main
+
+pool:
+  vmImage: ubuntu-latest
+
+jobs:
+- job: sbomManifestGenerator
+  steps: 
+  - checkout: self
+    persistCredentials: true
+
+  - task: CopyFiles@2
+    displayName: 'Copy Source directory to Target directory'
+    inputs:
+      sourceFolder: '$(Build.SourcesDirectory)'
+      Contents: |
+        **/*
+        !.git/**/*
+      targetFolder: '$(Build.ArtifactStagingDirectory)'
+
+  - task: Bash@3
+    displayName: 'Remove old Manifest files'
+    inputs:
+      targetType: 'inline'
+      script: |
+            cd $(Build.SourcesDirectory);
+            rm -rf _manifest;   
+
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 'Generate SBOM'
+    inputs:
+      BuildComponentPath: '$(Build.ArtifactStagingDirectory)'
+      BuildDropPath: '$(Build.SourcesDirectory)'
+      PackageName: 'Mission LZ'
+      PackageVersion: '$(Build.BuildNumber)'      
+    
+  
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: |
+            git config --global user.email $BUILD_REQUESTEDFOREMAIL
+            git config --global user.name "$BUILD_REQUESTEDFOR"
+            git checkout -t origin $(Build.SourceBranch);
+            git add '_manifest/*';
+            git status;
+            git commit -m "Software bill of materials changes"
+            git push --set-upstream origin HEAD:$(Build.SourceBranch);
```

Comment:
Some BASH nits

```suggestion
            git config --global user.email "$BUILD_REQUESTEDFOREMAIL"
            git config --global user.name "$BUILD_REQUESTEDFOR"
            git checkout -t origin $(Build.SourceBranch)
            git add '_manifest/*'
            git status
            git commit -m "Software Bill of Materials (SBOM) updated"
            git push --set-upstream origin HEAD:$(Build.SourceBranch)
```

Reply:
done

Reply:
Still seeing the previous commands in the latest changes in this branch